### PR TITLE
PBI AB#20349 [My Ministry > Members] [mobile] - Implement page details banner

### PR DESCRIPTION
**Pull request by IlyaRadinsky**
In this demo the `Expand` button showing in mobile mode only.

page-details-demo

@groberts314 , @morethanfire please have a look

### CODE REVIEW

**Comment by groberts314** [comment]
Is there any way to align the second row of statistics with the start of the first (leaving the space below the chart empty), like the mocks for My Ministry?

Members Page Details Interested Expanded

Members Page Details Mobile Expanded

File: docs/src/modules/pageDemo.js [hunk 1]
```diff
@@ -262,6 +262,47 @@ class PageDemo extends React.PureComponent {
...
+                        accessor: 'activeTemplates',
+                        header: 'Active Templates',
+                    }, {
+                        accessor: 'inactiveTemplates',
+                        header: 'Inactive Templates',
+                    },
+                ],
+                divide: true,
+            },
+        ];
```
Comment (groberts314) [inline docs/src/modules/pageDemo.js]: For demo purposes could we give the extra columns different names and different counts.  e.g. "Foo Templates" and "Bar Templates" maybe?  So we can tell that potentially different data is in the collapsible area?

**Comment by IlyaRadinsky** [comment]
@groberts314 yes, I saw the alignment problem, and we should decide how to resolve it (if it possible). The problem is that actually in demo page banner we have 3 columns:
- tittle (full width)
- stats column
- extra stats column

So, to align the `Expand` button with `stats column` there are should be in the same row and there are should be ability to pull right the `Expand` button. In my opinion, as the result, the config for columns will be tricky and complicated. So, after some thought, I've decided to present this variant - when all title/stats columns are in a single (and wrapable) column, and `Expand` button - is another one pulled right.

As another option, we could add `expandButtonMarginTop` prop for Page.Details.

**Comment by IlyaRadinsky** [comment]
page-details-demo2

**Comment by groberts314** [comment]
> So, after some thought, I've decided to present this variant - when all title/stats columns are in a single (and wrapable) column, and `Expand` button - is another one pulled right.

If I understand this option, correctly, this sounds like a good plan.  Will you be able to update this by tomorrow?  Did you need any further feedback from @morethanfire or me?

**Comment by IlyaRadinsky** [comment]
@groberts314 this is already done

**Comment by groberts314** [comment]
Oh, sorry, I thought it was still TODO.  Mainly because the screenshot still shows the second row left-aligned under the "chart" placeholder.  Do you have a screen shot of it with the desired alignment?

**Comment by IlyaRadinsky** [comment]
@groberts314 

Screen Shot 2019-10-09 at 7 40 30 PM

**Comment by groberts314** [comment]
image

**Comment by IlyaRadinsky** [comment]
Ah, I see... 
So, the alignment of `Not Contacted` with `1st Contact` we can get by adding the invisible stub for `extra stats column` (with the same size as the donut chart). The question is about margin top for `Expand` button. If it OK to add `expandButtonMarginTop` to Page.Details - I'll do

**Comment by groberts314** [comment]
> So, the alignment of Not Contacted with 1st Contact we can get by adding the invisible stub for extra stats column (with the same size as the donut chart).

Okay, got it, so this should be dealt with on consumer side when configuring the columns.  Makes sense.

> The question is about margin top for Expand button. If it OK to add `expandButtonMarginTop` to `Page.Details` - I'll do

@morethanfire How do you feel about `expandButtonMarginTop` prop?

**Comment by groberts314** [comment]
@IlyaRadinsky I'm hoping we're going to :shipit: on this one today shortly here, and release it in React CM UI `6.4.1`.  Assuming that's the case, will you be able to complete the HC side and submit a PR tomorrow?

**Comment by IlyaRadinsky** [comment]
@groberts314 yes, will do

File: src/modules/details.js [hunk 2]
```diff
@@ -147,6 +165,21 @@ class Details extends React.PureComponent {
                             );
                         })}
                     </div>
+                    {hasDetailedColumns && (
+                        <Button
+                            color="light"
+                            icon
```
Comment (groberts314) [inline src/modules/details.js]: We should probably add a `className` and/or `id` here.

**Pull request merged**